transport/mixer/src/delay.rs

```diff
@@ -0,0 +1,31 @@
+use std::time::Duration;
+
+pub const HOPR_MIXER_MINIMUM_DEFAULT_DELAY_IN_MS: u64 = 0;
+pub const HOPR_MIXER_DEFAULT_DELAY_DIFFERENCE_IN_MS: u64 = 200;
+
+/// Get a random delay duration from the specified minimum and maximum delay specified in milliseconds
+/// by the environment variables `HOPR_INTERNAL_MIXER_MINIMUM_DELAY_IN_MS` and `HOPR_INTERNAL_MIXER_MAXIMUM_DELAY_IN_MS`.
+pub fn random_delay() -> Duration {
+    let minimum = std::env::var("HOPR_INTERNAL_MIXER_MINIMUM_DELAY_IN_MS")
+        .map(|v| {
+            v.trim()
+                .parse::<u64>()
+                .unwrap_or(HOPR_MIXER_MINIMUM_DEFAULT_DELAY_IN_MS)
+        })
+        .unwrap_or(HOPR_MIXER_MINIMUM_DEFAULT_DELAY_IN_MS);
```

Comment:
Please make this a proper parameter, let it bubble up all the way to the top and let the executable binary set that from an env variable.